specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/examples/AnalyzeOperationResult.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{model Id}",
+    "resultId": "{result Id}",
+    "body": {}
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "status": "running",
+        "createdDateTime": "2019-05-01T10:53:21Z",
+        "lastUpdatedDateTime": "2019-05-01T10:53:23Z",
+        "analyzeResult": {
+          "version": "v2.0",
```

Comment:
given that we are going to GA 2.1 I would think we should return 2.1 for both 2.1-preview.1 and v2.1-preview.2 otherwise we might have to figure out what to return for v2.1 GA.